README.md

```diff
@@ -36,36 +36,43 @@ We all want to make sure our assignment actually performs _according to the requ
 
 ### Install Java
-_Puff_ uses a microservice architecture with [Spring Boot](https://spring.io/projects/spring-boot) written in Java.
+_Puff_ uses a microservice architecture with <a href="https://spring.io/projects/spring-boot" target="_blank">Spring Boot</a> written in Java
 
-* Download and install the [Java JDK 8](https://www.oracle.com/ca-en/java/technologies/javase/javase-jdk8-downloads.html)
+* Download and install the <a href="https://www.oracle.com/ca-en/java/technologies/javase/javase-jdk8-downloads.html" target="_blank">Java JDK 8</a>
 * Set the `JAVA_HOME` environment variable
 * Verify your installation by running `java -version`
 
 ### Install Maven
-_Puff_ uses [Maven](https://maven.apache.org/) as its build tool for its microservice backend.
-* [Download](https://maven.apache.org/download.cgi) and [install](https://maven.apache.org/install.html) Maven
+_Puff_ uses <a href="https://maven.apache.org/" target="_blank">Maven</a> as its build tool for its microservice backend.
+* <a href="https://maven.apache.org/download.cgi" target="_blank">Download</a> and <a href="https://maven.apache.org/install.html" target="_blank">Install</a> Maven
+* Make sure to add the bin directory of the created maven directory to your PATH
 * Verify your installation with `mvn -v`
 
 ### Setup Keycloak
 > Keycloak is an open source Identity and Access Management solution aimed at modern applications and services. It makes it easy to secure applications and services with little to no code.
 
-_Puff_ uses keycloak as a user management and authentication solution. More information about Keycloak can be found on their [offical docs page](https://www.keycloak.org/docs/latest/index.html).
+_Puff_ uses keycloak as a user management and authentication solution. More information about Keycloak can be found on their <a href="https://www.keycloak.org/docs/latest/index.html" target="_blank">offical docs page</a>.
 
-* Download and install the standalone server from https://www.keycloak.org/downloads.html.
+* Download and install the standalone server from <a href="https://www.keycloak.org/downloads.html" target="_blank">here</a>.
 * Navigate to your installed keycloak directory and run
 ```shell
 ./bin/standalone.sh -Djboss.socket.binding.port-offset=100
 ```
+* If you are using Windows instead run</dd>
+```shell
+./bin/standalone.bat -Djboss.socket.binding.port-offset=100
+```
 * Setup your admin account by navigating to http://localhost:8180.
 * Navigate to the admin portal from http://localhost:8180/auth/admin
+* When you log in you should be on the Master realm. Hover over the dropdown arrow and then click Add realm.
 * Import the realm settings, configuration and clients using the [puff-keycloak-config.json](security/puff-keycloak-config.json) file located in the security folder.
 
 #### Setup test account
 * Create two test user accounts and add one each to the `Users` and `Administrators` groups.
```

Comment:
Can you add that users are automatically part of the `Users` group. They should add at least one user to the `Administrators` group.